CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Dread
 
-- Fixed: The "Power Bomb Limitations" setting is now accounted for by logic.
+- Fixed: The "Power Bomb Limitations" setting is now accounted for by logic. For real this time.
```

Comment:
Might as well just immediately isolate this in 6.5 and move all the other changelog entries to 6.6

Reply:
Ok, so with Dakrszero's comment. I think it's reasonable to move this entry into the logic database section and rephrase it as "Fixed: The "Power Bomb Limitations" setting is now respected for opening Charge Beam Doors.".

Reply:
I think we should also put in a bit about enabling pb limitations again after it was disabled in 6.4.1, but it seems kind of odd to do that before stable gets merged back into main, because I don't know how the entire changelog will read.